src/bluetooth/BleService.js

```diff
@@ -373,6 +373,11 @@ export const getBleServiceInstance = manager => {
   if (!BleServiceInstance) {
     BleServiceInstance = new BleService(manager);
   }
+
+  if (manager) {
+    BleServiceInstance = new BleService(manager);
+  }
```

Comment:
added this to be able to update the service with a manager if needed.. think it shouldn't make much change?

Reply:
took me quick review to figure out how this worked with the intention of a singleton instance
to help a simpleton like me to use the singleton like this, a comment might help avoid confusion. maybe?
I had to check the usage, and saw that we only use without params, so is fine.
```
/**
 * Exports a singleton instance of the BleService
 * import the default and call without parameters to return the singleton
 * Call with a manager parameter to re-instantiate
 */

Reply:
Possibly better to add a field `setManager` on the class?

Reply:
53b8542a5869aabbe799e6012d4b90aee3c53314